strip.lua

```diff
@@ -227,6 +227,9 @@ function trunkReturn()
     else
         state.currentState = "unload"
     end
+    if checkFuel() == false then
```

Comment:
This shouldn't be here. You will want outOfFuel to be a state that decideState can go to